raiden/tests/fixtures/blockchain.py

```diff
@@ -226,9 +201,12 @@ def cached_genesis(request, blockchain_type):
 
         genesis_alloc[account_address] = account_alloc
 
+    all_keys = list(private_keys)
+    all_keys.append(deploy_key)  # needs to be at the end because of the minerthreads keys
```

Comment:
The all_keys collection is sorted in the next line, so the deploy key is not necessarily the last one